package.json

```diff
@@ -29,6 +29,7 @@
   "dependencies": {
     "axios": "^0.21.1",
     "babel-loader": "^8.2.2",
+    "body-parser": "^1.19.0",
```

Comment:
Remove body-parser as it's deprecated. Instead use express.json() as middleware when needed. or express.urlencoded()
Additional configs: https://expressjs.com/en/api.html#express.json